.gitignore

```diff
@@ -12,3 +12,4 @@ wiki
 param.log
 latex
 html
+.vs*
```

Comment:
why do you need to add this to the gitignore? I don't see a new feature in your PR which adds these kind of files.